src/app/modules/main/wallet_section/activity/controller.nim

```diff
@@ -244,6 +244,9 @@ QtObject:
       let args = SavedAddressArgs(e)
       self.model.refreshItemsContainingAddress(args.address)
 
+    events.on(SIGNAL_TOKENS_LIST_UPDATED) do(e:Args):
+      self.model.refreshAmountCurrency(self.currencyService)
```

Comment:
@Cuteivist is it really needed here or we can set the amount appropriately when setting entries (`setEntries` and `updateEntries`) for the model?

Reply:
Not sure that I got you, what's heavier? 
In that implementation we're refreshing every time `SIGNAL_TOKENS_LIST_UPDATED` is emitted.

What I meant is to refresh the amount when items are added to the model, like here https://github.com/status-im/status-desktop/blob/8d8268db2ff1b632211a90beb5ad997fce0b0171/src/app/modules/main/wallet_section/activity/controller.nim#L118 and here https://github.com/status-im/status-desktop/blob/8d8268db2ff1b632211a90beb5ad997fce0b0171/src/app/modules/main/wallet_section/activity/controller.nim#L260

But when think about it, it must be up to some broken dependencies on the qml side, cause it must be displayed well when new item is added.